lib/rack/multipart/generator.rb

```diff
@@ -69,12 +73,13 @@ def flattened_params
       end
 
       def content_for_tempfile(io, file, name)
+        length = ::File.stat(file.path).size if file.path
+        filename = "; filename=\"#{Utils.escape(file.original_filename)}\"" if file.original_filename
```

Comment:
Does this create a regression? Filenames with spaces will now have a `+` instead of the space.

Does that means that clients will need to unescape de filename prior to use?